src/lib/edge-functions/editor-helper.js

```diff
@@ -7,6 +7,7 @@ const { runRecipe } = require('../../commands/recipes')
 const STATE_PROMPT_PROPERTY = 'promptVSCodeSettings'
 
 const promptEditorHelper = async ({ NETLIFYDEVLOG, chalk, config, log, repositoryRoot, state }) => {
+  if (env.NODE_ENV === 'test') return
```

Comment:
I think it'd be good to add a comment to clarify the intent here.

```suggestion
  // This prevents tests from hanging when running them inside the VS Code
  // terminal, as otherwise we'll show the prompt and wait for a response.
  if (env.NODE_ENV === 'test') return
    
```